warp10/src/main/java/io/warp10/continuum/ingress/Ingress.java

```diff
@@ -1545,6 +1545,22 @@ public void handleDelete(String target, Request baseRequest, HttpServletRequest
         throw new IOException("Parameter " + Constants.HTTP_PARAM_DELETEALL + " should be set when deleting a full range.");
```

Comment:
When the `nodata` (or deletemeta) option is set, we should consider it's correspond to a `deleteall`call and not throw this error.